### PR TITLE
chore(drun): Disable DTS for deterministic debug outputs

### PR DESCRIPTION
**Pull request by luc-blaeser**
Disable DTS in `drun` for system subnets to enable deterministic debug output testing.

**Background information**:
DTS aborts uncompleted messsages if they reach a checkpoint and retries them later. However, debug prints of such DTS-aborted executions leak, which leads to non-deterministic debug outputs and can disturb testing with `drun`. Therefore, disable DTS on system subnets in `drun` for deterministic debug outputs as required by Motoko's tests.

This testing issue has become noticeable with https://github.com/dfinity/ic/pull/621.

### CODE REVIEW

**Review by ggreif — APPROVED** [review]
LGTM!

**Comment by luc-blaeser** [comment]
Thank you, Gabor!

**Review by crusso — APPROVED** [review]
Hmm, I wonder if those replicated effects make it into the logs!

File: rs/drun/src/lib.rs [hunk 1]
```diff
@@ -185,6 +185,14 @@ pub async fn run_drun(uo: DrunOptions) -> Result<(), String> {
         cfg.hypervisor.max_query_call_graph_instructions = instruction_limit;
     }
 
+    // DTS aborts uncompleted messsages if they reach a checkpoint and retries them
+    // later. However, debug prints of such DTS-aborted executions leak, which leads
+    // to non-deterministic debug outputs and disturbs testing.
+    // Therefore, disable DTS on system subnets for deterministic debug outputs.
```
Comment (crusso) [inline rs/drun/src/lib.rs:191]: Maybe create an issue to fix this when debug prints are implemented properly.

**Review by crusso — APPROVED** [review]

**Comment by luc-blaeser** [comment]
Just checked: The output of DTS-aborted messages are not written to the log. So, just a local testing phenomenon.

**Pull request merged**